skyportal/handlers/api/photometry.py

```diff
@@ -380,14 +393,12 @@ def serialize(
                 "limiting_mag": maglimit_out,
             }
 
-            if extinction:
+            if extinction_dict is not None:
```

Comment:
Since you now check if the dict is not None in all your "ifs" except the first one where you also check if `extinction` param is `True`, do we even need that extinction boolean parameter? Best to remove it if we anyways have extinction_dict as a proxy, that's already what you use most of the time